docs/examples/sheetmobile/withPrimaryActionHeader.js

```diff
@@ -0,0 +1,51 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import {
+  Layer,
+  SheetMobile,
+  Box,
+  TextField,
+  DeviceTypeProvider,
+  Button,
+  FixedZIndex,
+  CompositeZIndex,
+} from 'gestalt';
+
+export default function Example(): Node {
+  const [showComponent, setShowComponent] = useState(true);
+  const PAGE_HEADER_ZINDEX: FixedZIndex = new FixedZIndex(10);
+  const ABOVE_PAGE_HEADER_ZINDEX: CompositeZIndex = new CompositeZIndex([PAGE_HEADER_ZINDEX]);
+
+  return (
+    <DeviceTypeProvider deviceType="mobile">
+      {showComponent ? (
+        <Layer zIndex={ABOVE_PAGE_HEADER_ZINDEX}>
+          <SheetMobile
+            align="center"
+            heading="Create a new personal account"
+            onDismiss={() => setShowComponent(false)}
+            showDismissButton={false}
+            primaryAction={{ accessibilityLabel: 'Next page', label: 'Next', onClick: () => {} }}
+          >
+            <TextField
```

Comment:
Can we add a footer to this, so we can test how the footer behaves with an input

Reply:
DONE